tests/blackbox/stratisd_cert.py

```diff
@@ -477,6 +477,48 @@ def test_filesystem_create_permissions(self):
 
         self._test_permissions(StratisDbus.fs_create, [pool_path, fs_name], True)
 
+    def test_filesystem_udev_symlink(self):
+        """
+        Test the udev symlink creation for filesystem devices.
+        """
+        pool_name = p_n()
+        pool_path = make_test_pool(pool_name, StratisCertify.DISKS[0:1])
+
+        fs_name = fs_n()
+        filesystem_path = make_test_filesystem(pool_path, fs_name)
+
+        objects = StratisDbus.get_managed_objects()
+
+        pool_gmodata = objects[pool_path]
+        pool_uuid = pool_gmodata["org.storage.stratis2.pool"]["Uuid"]
```

Comment:
The interface names should be constants, and I think they should probably be the constants that correspond to the most recent interface revision. Probably the best way to do this is to make the necessary constants public in the StratisDbus class. Properly speaking, you should remove the underscores when you make them public.

Reply:
I see these in tests/blackbox/testlib/dbus.py:
```
    _POOL_IFACE = "org.storage.stratis2.pool.r1"
    _FS_IFACE = "org.storage.stratis2.filesystem"
```

Reply:
OK; the diffstat for the change that would be necessary to make `_FS_IFACE` and `_POOL_IFACE` public will be about 60 lines in tests/blackbox/testlib/dbus.py.

I'm going to be rebasing, and adding the commits for this change, and removing the escaping that is no longer necessary.

Reply:
Resolved.